src/components/ProgressBar/ProgressBar.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import styles from './ProgressBar.module.scss';
+
+export type ProgressBarPropsT = {
+  value: number;
+  classProp?: string;
+};
+
+const ProgressBar = ({ value = 40, classProp = '' }: ProgressBarPropsT) => {
+  return (
+    <div className={`${styles.wrapper} ${classProp}`}>
+      <div className={styles.container}></div>
+      <div className={`${styles.value} progress_value`} style={{ width: value + '%' }}></div>
```

Comment:
Why does this have a hard-coded "progress_value" class?

Reply:
It's to give access to the "value" div. In the new design it changes color quite a bit. We have a lot of abstraction with color vars right now so I feel like a class to pinpoint to change the color is safer than passing a color hex.

But now looking back at it.. it might be better to pass a classValueProp and just add that to the props to inject a class right on to the value....? maybe thats better..

Here is an impimentation of the current on turkey..
<img width="420" alt="Screen Shot 2022-07-29 at 9 19 58 AM" src="https://user-images.githubusercontent.com/10427235/181802104-0e98e3bc-31c5-4e96-a0d5-34c573a85900.png">